contributing.md

```diff
@@ -27,31 +53,20 @@ yarn prepublish
 
 If you need to clean the project for any reason, use `yarn clean`.
 
-## Adding warning/error descriptions
-
-In Next.js we have a system to add helpful links to warnings and errors.
-
-This allows for the logged message to be short while giving a broader description and instructions on how to solve the warning/error.
-
-In general all warnings and errors added should have these links attached.
-
-Below are the steps to add a new link:
-
-- Create a new markdown file under the `errors` directory based on `errors/template.md`: `cp errors/template.md errors/<error-file-name>.md`
-- Add the newly added file to `errors/manifest.json`
-- Add the following url to your warning/error: `https://nextjs.org/docs/messages/<file-path-without-dotmd>`. For example to link to `errors/api-routes-static-export.md` you use the url: `https://nextjs.org/docs/messages/api-routes-static-export`
+## Testing
 
-## To run tests
-
-Make sure you have `chromedriver` installed for your Chrome version. You can install it with
+Make sure you have `chromedriver` installed, and it should match your Chrome version.
+You can install it with:
 
+- `apt install chromium-chromedriver` on Ubuntu/Debian
```

Comment:
i think on debian it's actually `chromium-driver`. see https://packages.debian.org/bullseye/chromium-driver

Reply:
I tested this directly—`chromium-chromedriver` was the most recommended package in search results and seems to be maintained by the Ubuntu MOTU devs.

It installs standalone Chromium and then the corresponding chromedriver.

https://packages.ubuntu.com/bionic/chromium-chromedriver

Reply:
afaik ubuntu does not follow debian in this case, because the chromium package on debian is distributed as a `snap` package

Reply:
OK, we can change it. Good note.